maplibre-gl-js-vue-amplify-compare-maps/vite.config.js

```diff
@@ -0,0 +1,15 @@
+import { fileURLToPath, URL } from 'url';
+
+import { defineConfig } from 'vite';
+import vue from '@vitejs/plugin-vue';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [vue()],
+  resolve: {
+    alias: {
+      '@': fileURLToPath(new URL('./src', import.meta.url)),
+      './runtimeConfig': './runtimeConfig.browser',
+    },
+  },
```

Comment:
Please add the config [from here](https://github.com/aws-samples/amazon-location-samples/blob/main/maplibre-gl-js-vue-amplify/vite.config.js#L14-L17).

Reply:
Done